about.html

```diff
@@ -0,0 +1,139 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+    <link rel="stylesheet" href="abtt.css">
+    <title>Bole Festival 2023 About page</title>
+</head>
+
+<body>
+    <header>
+        <section class="header" id="header">
+            <nav class="nav-container1" id="nav-container1">
+                <div class="nav-menu1" id="nav-menu1">
+                    <ul class="nav-list1" id="">
+                        <li class="nav-item1"><a href="#">
+                                <i class="ri-facebook-fill"></i>
+                            </a></li>
+                        <li class="nav-item1"><a href="#">
+                                <i class="ri-twitter-fill"></i>
+                            </a></li>
+                        <li class="nav-item1"><a href="#">Register</a></li>
+                        <li class="nav-item1"><a href="#">Connect</a></li>
+                        <li class="nav-item1"><a href="#">Donate</a></li>
+                    </ul>
+                </div>
+            </nav>
+            <nav class="nav-container2 flex-nav" id="nav-container2">
+                <div class="nav-toggle" id="nav-toggle">
+                    <i class="ri-menu-fill"></i>
+                </div>
+                <ul class="nav-list2" id="nav-list2">
+                    <i class="ri-close-fill"></i>
+                    <li class="clicklogo"><a href="#"><img src="./img/bolefest.png" alt="logo"></a></li>
+                    <li class="click"><a href="./index.html">Home</a></li>
+                    <li class="click"><a href="#">Sponsors</a></li>
+                    <li class="click"><a href="#">News</a></li>
+                    <li class="clicks"><a href="#">FF Campaign</a></li>
+                </ul>
+            </nav>
+        </section>
+        <div class="sectiontwo">
+            <section2 class="homepage">
+                <h3>
+                    Back Bigger Better ...
+                </h3>
+                <h1>Bole Community Festival 2023!!! <br></h1>
+                <p>
+                    What happens at Bole Festival?
+                    Bole festival is a place where the southern food lovers converge to eat “Bole” the beloved and
+                    most popular street food in the South. We also create a safe and fun space for food lovers to
+                    meet food vendors. We always provide an assortment of fun games and competitions.
+                </p>
+            </section2>
+            <section class="enq">
+                <div class="enquire-container">
+                    <p class="enquries">Please contact us per Email for any further quetions about Bole
+                        Festival 2023</p>
+                    <p><a class="about-email" href="mailto: chizzyjiver@yahoocom">chizzyjiver@yahoo.com</a></p>
+                </div>
+            </section>
+        </div>
```

Comment:
- [ ]  You about page should contains a middle section with a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo.

Your design 

![image](https://user-images.githubusercontent.com/43877689/227662882-bae00ffe-7e46-495c-8f31-3bf14a1d6e9c.png)

Behance design 
![image](https://user-images.githubusercontent.com/43877689/227662932-d86cbb25-76c2-4e5b-b8f3-1177eb94439f.png)